### DISCUSSION TITLE
retext-sentiment deprication

### DISCUSSION THREAD

**Discussion by tvquizphd** [retextjs/retext#71] [Q&A] [answered]
Hi @wooorm! I'd like to politely ask about [retext-sentiment](https://github.com/retextjs/retext-sentiment)'s deprecation in favor of [polarity](https://github.com/words/polarity). Is there any particular functionality of that plugin you felt a need to deprecate, or is the plugin simply no longer a priority?

**Comment by wooorm** [comment, marked as answer]
Hey!

Not a priority — nobody is using it.
It was fun to create yeaaars ago but as I’m maintaining hundreds of projects already and seriously pressed for time maintaining it all, it feels like a waste to keep it going actively!

**Reply by tvquizphd** [reply]
Understandable, thank you for such a fast response!
